core/templates/dev/head/pages/privacy/privacy.html

```diff
@@ -3,7 +3,11 @@
 {% block maintitle %}
   Privacy Policy - Oppia
 {% endblock maintitle %}
-
+{% block header_css %}
+  {{ super() }}
+  <link rel="stylesheet" type="text/css" media="screen"
```

Comment:
It seems a bit weird that non-About pages are using about.css. If the About classes are being used in multiple pages then presumably they are misnamed and we should change the naming convention to refer to something more general?

More likely, I suspect there's some stuff being used across multiple pages, and some stuff that's only part of the About page. We should have different policies for dealing with each of these things. The former might go in oppia.css since it's a general file; the latter could go in an About-page-specific CSS file or even be included as an inline style tag.

Reply:
This sounds good! There are a few such classes being used across multiple pages. Can you please suggest a naming convention for them?  
Thanks!

Reply:
I don't know; it depends on what pages they're shared across. Perhaps you could figure out which pages those are and propose an appropriate name.

Reply:
So we can go with the convention: oppia-<page 1>-<page 2>-<seletor name>   
For example, if the `tabs` selector is being used in `about.html` as well as`teach.html`, we can rename it to `oppia-about-teach-tabs`. How does this sound?

Reply:
That's a possibility, yep! We just have to ensure that we don't use such classes in other pages.

An alternative would be to find a collective noun that describes both the About and Teach pages (e.g. oppia-info-pages-*) and use that. That might scale better if we add more pages with the same layout and structure.

Reply:
I think finding a unique collective noun that does not coincide with other classes might be a bit difficult. Also, it would be difficult to understand which collective noun refers to which page(s).  
For example, some classes are common for `about.html` and `teach.html`, some for `about.html` and `privacy.html` and some for all the three. Here, using info-pages can refer to any of those pages.  
With the other approach, we can ensure uniqueness but scalability might be a problem.  
We can combine both of them: for example, we can use individual page names up to say 3 pages and then for classes in all four pages (here), we can use a collective noun and comment it down before the selector in the CSS file where it is used.  
How does this sound?

Reply:
Yes, that sounds like a reasonable convention! Let's migrate a few files to have CSS like this, to make sure it works in practice, and then we can document it in the style guide.

Reply:
Okay, thanks!

Reply:
Also, another peculiar thing: I found out that a class, `oppia-about-anchor` is not being used in `about.html`. It is only being used in `privacy.html` and `terms.html`.

Reply:
That's fine, that just means it needs renaming.

Reply:
Okay, thanks!